ext/RMagick/rmimage.c

```diff
@@ -10914,6 +10914,7 @@ Image_polaroid(int argc, VALUE *argv, VALUE self)
 
     exception = AcquireExceptionInfo();
 #if defined(IMAGEMAGICK_7)
+    caption = GetImageProperty(clone, "Caption", exception);
```

Comment:
Maybe we could also add a caption argument and set that for the IM6 path?

Reply:
Good idea :)